src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectParameterNames.cs

```diff
@@ -51,6 +53,56 @@ public static class OpenIdConnectParameterNames
         public const string UserId = "user_id";
         public const string Username = "username";
         public const string VersionTelemetry = "x-client-ver";
-#pragma warning restore 1591
     }
+
+    /// <summary>
+    /// Parameter names for OpenIdConnect UTF8 bytes.
+    /// </summary>
+    public static class OpenIdConnectParameterUtf8Bytes
```

Comment:
does this need to be public or can it be internal?

Reply:
It's a tossup, we make the parameters public and some users like that.

For now let's make it internal.